mne/io/meas_info.py

```diff
@@ -1085,6 +1080,7 @@ def read_meas_info(fid, tree, clean_bads=False, verbose=None):
         elif kind == FIFF.FIFF_MNE_KIT_SYSTEM_ID:
             tag = read_tag(fid, pos)
             kit_system_id = int(tag.data)
+    rename = _read_extended_ch_info(chs, meas_info, fid)
```

Comment:
the name `rename` suggests it should be a bool. It took me a bit of time
to understand that what you call rename here corresponds to the `mapping`
parameter in `rename_channels`. Would ch_names_mapping be too long?